app/logout.go

```diff
@@ -15,17 +18,21 @@ func NewLogoutCommand(c *LoginClient) (CommandOut, error) {
 			disablePopUpFlag,
 		},
 		Action: func(ctx *cli.Context) error {
-			if err := c.loginService.DeleteConfigFile(getTokenConfigPath(ctx)); err != nil {
+			configDir := ctx.Path(ConfigDirFlagName)
```

Comment:
should we still try to logout even in this error case?

Reply:
Oh yes we should.